packages/curve-ui-kit/src/lib/queries/factory.ts

```diff
@@ -1,6 +1,5 @@
 import type { UseQueryOptions } from '@tanstack/react-query'
 import { QueryKey, useQuery } from '@tanstack/react-query'
-import { queryClient } from '@ui-kit/lib/api/query-client'
```

Comment:
Note to self: prob will need it back for a `createFetchQuery` that has to be used in slices which can't use `useQueryClient()`